src/docs/user/references/DataConnectors.adoc

```diff
@@ -318,7 +318,10 @@ Default: `HTML`
 
 The scheduler should be configured in non-fork mode to execute tasks in a single JVM rather than starting a dedicated JVM to run the task (In $PROACTIVE_HOME/config/scheduler/settings.ini, set pa.scheduler.task.fork=false).
 
-To be executed, each workflow requires the jdbc connector jar file of the used rdbms to connect to the database. Make sure you have it properly copied into the addons directory ($PROACTIVE_HOME/addons) of the scheduler.
+To be executed, each workflow requires the jdbc connector jar of the used rdbms to enable connection to the database.
+In case the Scheduler is running with only local Nodes, make sure that you have properly added the jdbc jar into the addons directory ($PROACTIVE_HOME/addons) of the Scheduler before starting the server.
```

Comment:
is running with only local Nodes => is running on local Nodes only
make sure that you have properly added the jdbc jar into the addons directory ($PROACTIVE_HOME/addons) of the Scheduler before starting the server. => make sure that you have properly added the jdbc jar, before starting the server, into the addons directory ($PROACTIVE_HOME/addons) of the Scheduler.